src/main/containers/TestPanel.jsx

```diff
@@ -1,11 +1,24 @@
 import React, { useState } from 'react';
+import ViewTest from '../components/ViewTest.jsx';
+
+// will need to get data from the get request to pass to the formatted view
 
 const TestPanel = (props) => {
   const [mockups, setData] = useState([props.data]);
+  // sample JSON to pass down as props. Will be able to remove as the project evolves.
+  const data = {
```

Comment:
A little confused—is the thought here that we’ll pass `data` as a prop from the source panel to this panel? And this declaration is just for testing?

Reply:
it is, this just testing the data, below you can see its passed in as a prop on line 21